### PR TITLE
Microverse_portfolio

### PR DESCRIPTION
**Pull request by vaderonline**
Added About Section, Work Section, Updated Readme 
By using Grid.
The CSS Grid Layout Module offers a grid-based layout system, with rows and columns, making it easier to design web pages without having to use floats and positioning
[ScreenShots](https://drive.google.com/file/d/1ScrE_KxmGuK2iSB81CFM5uazy7nTuLOs/view?usp=sharing)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +21,8 @@ To get a local copy up and running follow these simple example steps.
...
 <h2 dir="auto"><a id="user-content-built-with" class="anchor" aria-hidden="true" href="#built-with"><svg class="octicon octicon-link" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"></path></svg></a>👤 Author1</h2>
 <ul>
-  <li>GitHub:<a href="https://github.com/vaderonline">vaderonline!  &nbsp; Hire !</a></li>
-  <li>Linkedin:<a href="https://www.linkedin.com/in/vishal-kumar-514455152/">Vishal kumar &nbsp; Hire !</a></li>
+  <li>GitHub:<a href="https://github.com/vaderonline">vaderonline!  &nbsp; <mark>contact me </mark></a></li>
+  <li>Linkedin:<a href="https://www.linkedin.com/in/vishal-kumar-514455152/">Vishal kumar &nbsp;<mark> Contact me</mark</a></li>
   </ul>
```
Comment (uwadonat) [inline README.md]: - [ ] Kindly fill all missing information and correct typo errors in your Readme file. you can follow this [template](https://github.com/microverseinc/readme-template/blob/master/README.md)
- [x] Make sure all your sections have the proper background image (decorations are always backgrounded images, not img tags)

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[vaderonline](https://github.com/vaderonline),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost

## Highlights
- No linter error ✔️
- Your are following HTML/CSS best practice ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (vaderonline) [reply README.md]: Thanks for reviewing my project as you mention about  the background image I used the same background image as the template have [figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471)

**Review by DeliceLydia — APPROVED** [review]
## Hi @vaderonline, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**